test/test_informationgain.jl

```diff
@@ -34,12 +34,25 @@ end
 end
 
 # Test cases for best_split
-@testset "Best Split" begin
- 
-
+@testset "best_split" begin
     X = [1.0 2.0; 3.0 4.0; 2.5 0.5; 4.0 3.0]
     y = [0, 1, 0, 1]
     best_feature, best_threshold = DecisionTreeAndRandomForest.best_split(X, y)
     @test best_feature == 1
-    @test best_threshold == 2.5
+    @test best_threshold == 3.0
+end
+
+@testset "split_gini_categorical" begin
```

Comment:
This should be called split_ig_categorical

Reply:
Yes just noticed it as well. Was about to change it now :)